src/components/dynamic/admin/dashboards/Toolbar.jsx

```diff
@@ -139,12 +139,17 @@ const Toolbar = ({
       <div className="flex w-1/3">
         <FaTrashAlt
           data-cy="delete"
-          onClick={() =>
-            setPopup({
-              ...popup,
-              visible: true,
-            })
-          }
+          onClick={() => {
+            const numSelectedObjects = objects.filter((a) => a.selected).length;
+            if (numSelectedObjects) {
```

Comment:
should be an explicit number check such as `objects.filter((a) => a.selected).length === 0`